app/rest/model/src/main/java/io/syndesis/model/integration/IntegrationStatus.java

```diff
@@ -28,13 +28,13 @@
      * The desired state of the revision.
      * @return
      */
-    IntegrationRevisionState getTargetState();
+    IntegrationDeploymentState getTargetState();
 
     /**
      * The current state of the revision.
      * @return
      */
-    IntegrationRevisionState getCurrentState();
+    IntegrationDeploymentState getCurrentState();
 
```

Comment:
If we want to track draft status, perhaps we should add it here:  boolean isDraft()

Reply:
I'll remove this too. Was proposed by rhuss in July, but most of the have moved to other objects and this object is not really used.